### ISSUE TITLE
Add new Map module to add Location .

### ISSUE THREAD

**Issue by sagarpreet-chadha** [publiclab/PublicLab.Editor#210]
### **Mock up** : 

![image](https://user-images.githubusercontent.com/14952645/50899692-e25d4c00-1438-11e9-8b20-be74bdcaa212.png)

![image](https://user-images.githubusercontent.com/14952645/50899695-e5583c80-1438-11e9-83fc-79a45f4eac62.png)

**Comment by geekychaser** [comment]
@sagarpreet-chadha @jywarren can i give it try ?
Thank you

**Comment by jywarren** [comment]
Yes please! Be aware this may be a somewhat complex task - to add a new map
module -- but we can take it one step at a time. Upon selecting a location,
we should append latitude/longitude tags to the tag input!

We can start by having the editor pop up a modal just like on the tag area
of pages like this: https://publiclab.org/wiki/micro

except that it adds the tags to the form, rather than to an already
existing page.

We can add a map display feature as a follow-up - let's do this in small
progressive parts!

**Comment by sagarpreet-chadha** [comment]
@geekychaser ...i have already started this issue (hopefully completed now  , waiting for @jywarren 🐵  for review) . 
I am currently trying to solve the errors shown by travis , maybe you can help ❓ . I would love 😃  to collaborate with you on another issue 💃 .
Thank you !
